packages/inter-protocol/test/swingsetTests/reserve/bootstrap-assetReserve-upgrade.js

```diff
@@ -0,0 +1,264 @@
+
+const moola = withAmountUtils(makeIssuerKit('moola'));
+
+export const buildRootObject = async () => {
+  const storageKit = makeFakeStorageKit('assetReserveUpgradeTest');
+  const { nameAdmin: namesByAddressAdmin } = makeNameHubKit();
+  const timer = buildManualTimer();
+  const marshaller = makeBoard().getReadonlyMarshaller();
+
+  const { promise: committeeCreator, ...ccPK } = makePromiseKit();
+
+  /** @type {ZoeService} */
+  let zoeService;
+
+  /** @type {FeeMintAccess} */
+  let feeMintAccess;
+
+  /** @type {Subscriber<import('../../../src/reserve/assetReserveKit.js').MetricsNotification>} */
+  let metrics;
+  /** @type {UpdateRecord<import('../../../src/reserve/assetReserveKit.js').MetricsNotification>} */
+  let metricsRecord;
+
+  /** @type {VatAdminSvc} */
+  let vatAdmin;
+
+  let initialPoserInvitation;
+  let poserInvitationAmount;
+
+  // for startInstance
+  /**
+   * @type {{
+   * committee?: Installation<import('@agoric/governance/src/committee').start>,
+   * assetReserveV1?: Installation<import('../../../src/reserve/assetReserve').prepare>,
+   * puppetContractGovernor?: Installation<import('@agoric/governance/tools/puppetContractGovernor').start>,
+   * }}
+   */
+  const installations = {};
+
+  /** @type {import('@agoric/governance/tools/puppetContractGovernor').PuppetContractGovernorKit<import('../../../src/reserve/assetReserve.js').prepare>} */
+  let governorFacets;
+  /** @type {ReturnType<Awaited<ReturnType<import('../../../src/reserve/assetReserve.js').prepare>>['creatorFacet']['getLimitedCreatorFacet']>} */
+  let arLimitedFacet;
+
+  /** @type {Omit<import('@agoric/zoe/src/zoeService/utils.js').StartParams<import('../../../src/reserve/assetReserve.js').prepare>['terms'], 'issuers' | 'brands'>} */
+  const arTerms = {
+    governedParams: {
+      // @ts-expect-error missing value
+      [CONTRACT_ELECTORATE]: {
+        type: ParamTypes.INVITATION,
+      },
+    },
+  };
+  const staticPrivateArgs = {
+    storageNode: storageKit.rootNode,
+    marshaller,
+    namesByAddressAdmin,
+  };
+
+  return Far('root', {
+    bootstrap: async (vats, devices) => {
+      vatAdmin = await E(vats.vatAdmin).createVatAdminService(devices.vatAdmin);
+      ({ feeMintAccess, zoeService } = await E(vats.zoe).buildZoe(
+        vatAdmin,
+        undefined,
+        'zcf',
+      ));
+
+      const v1BundleId = await E(vatAdmin).getBundleIDByName(arV1BundleName);
+      v1BundleId || Fail`bundleId must not be empty`;
+      installations.assetReserveV1 = await E(zoeService).installBundleID(
+        v1BundleId,
+      );
+
+      installations.puppetContractGovernor = await E(
+        zoeService,
+      ).installBundleID(
+        await E(vatAdmin).getBundleIDByName('puppetContractGovernor'),
+      );
+
+      installations.committee = await E(zoeService).installBundleID(
+        await E(vatAdmin).getBundleIDByName('committee'),
+      );
+      const ccStartResult = await E(zoeService).startInstance(
```

Comment:
what's "cc"?   "ec" would have made some sense to me.

Reply:
committee contract, I guess. it's the third instance of this so I'll leave it for consistency